.gitlab-ci.yml

```diff
@@ -254,58 +254,58 @@ generate-impl-guide:
       # https://docs.gitlab.com/ee/ci/multi_project_pipelines.html#with-variable-inheritance
       dotenv: ./artifacts/build.env
 
-publish-polkadot-image:
-  stage:                           build
-  <<:                              *build-push-image
-  variables:
-    <<:                            *image-variables
-    # scripts/docker/Dockerfile
-    DOCKERFILE:                    Dockerfile
-    IMAGE_NAME:                    docker.io/paritypr/synth-wave
-  rules:
-  # Don't run on releases - this is handled by the Github Action here:
-  # .github/workflows/publish-docker-release.yml
-    - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
-      when: never
-    - if: $CI_PIPELINE_SOURCE == "schedule"
-    - if: $CI_COMMIT_REF_NAME == "master"
-    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
-    - if: $CI_COMMIT_REF_NAME == "rococo-v1"
-      variables:
-        <<:                        *image-variables
-        IMAGE_NAME:                docker.io/parity/rococo
-        DOCKER_USER:               ${Docker_Hub_User_Parity}
-        DOCKER_PASS:               ${Docker_Hub_Pass_Parity}
-  needs:
-    - job:                         test-build-linux-stable
-      artifacts:                   true
+# publish-polkadot-image:
+#   stage:                           build
+#   <<:                              *build-push-image
+#   variables:
+#     <<:                            *image-variables
+#     # scripts/docker/Dockerfile
+#     DOCKERFILE:                    Dockerfile
+#     IMAGE_NAME:                    docker.io/paritypr/synth-wave
+#   rules:
+#   # Don't run on releases - this is handled by the Github Action here:
+#   # .github/workflows/publish-docker-release.yml
+#     - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
+#       when: never
+#     - if: $CI_PIPELINE_SOURCE == "schedule"
+#     - if: $CI_COMMIT_REF_NAME == "master"
+#     - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
+#     - if: $CI_COMMIT_REF_NAME == "rococo-v1"
+#       variables:
+#         <<:                        *image-variables
+#         IMAGE_NAME:                docker.io/parity/rococo
+#         DOCKER_USER:               ${Docker_Hub_User_Parity}
+#         DOCKER_PASS:               ${Docker_Hub_Pass_Parity}
+#   needs:
+#     - job:                         test-build-linux-stable
+#       artifacts:                   true
```

Comment:
I'm not sure if this job is related to simnet, looks like we used it previously to publish rococo-v1 images among other things.

Reply:
AFAIK @TriplEight [has introduced it](https://github.com/paritytech/polkadot/pull/2643/commits/a4420dff5f1c49aff3b29c42de051363f57592fe#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R273) specifically for the simnet's needs. Anyway, I'll re-enable it as soon as I'll fix the new runner setup for such cases.

Reply:
I'm referring to
```
#     - if: $CI_COMMIT_REF_NAME == "rococo-v1"
#       variables:
#         <<:                        *image-variables
#         IMAGE_NAME:                docker.io/parity/rococo
#         DOCKER_USER:               ${Docker_Hub_User_Parity}
#         DOCKER_PASS:               ${Docker_Hub_Pass_Parity}
#   needs:
#     - job:                         test-build-linux-stable
#       artifacts:                   true
```
part, which was introduced before that. I think we still want to publish rococo-v1 binaries as it was requested by external teams trying to test and deploy parachains.

Let's comment out only simnet part then and reeable it together with the simnet check once it's ready.